addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -51,13 +51,13 @@ const Centered = styled.span<{}>({
 type Status = 'initial' | 'manual' | 'running' | 'error' | 'ran' | 'ready';
 
 export const A11YPanel: React.FC = () => {
-  const [status, setStatus] = useAddonState<Status>(ADDON_ID, 'initial');
-  const [error, setError] = React.useState<unknown>(undefined);
-  const { setResults, results } = useA11yContext();
-  const { storyId } = useStorybookState();
   const { manual } = useParameter<Pick<A11yParameters, 'manual'>>('a11y', {
     manual: false,
   });
+  const [status, setStatus] = useState<Status>(manual ? 'manual' : 'initial');
```

Comment:
Using useAddonState on this didn't worked
- useEffect doesn't update 'right' addon state
- adding it to dependencies of useEffect overflows